src/test/java/com/codurance/retropolis/acceptance/cards/AddUpVoteStepDefinitionIntegrationTest.java

```diff
@@ -38,7 +37,7 @@ public void theClientUpdatesToCardsWithThisIdInPathAndAddUpVoteInBody(String use
     Card card = new ObjectMapper().readValue(responseResult.getBody(), new TypeReference<>() {
     });
     executePatch("http://localhost:5000/cards/" + card.getId() + "/vote",
```

Comment:
Can you please use the `url` field from the parent class instead of hardcoding the `http://localhost:5000/cards/`?